run.py

```diff
@@ -1,9 +1,70 @@
+import argparse
+import logging
 import os
+from distutils.util import strtobool
+
 from behave import __main__ as behave_executable
 
+logger = logging.getLogger(__name__)
 
-if __name__ == '__main__':
-    format = os.getenv('BEHAVE_FORMAT', 'progress2')
+DEFAULT_COMMAND_LINE_ARGS = '--stop'
+DEFAULT_BEHAVE_FORMAT = 'progress2'
+DEFAULT_ACCEPTANCE_FEATURES_DIRECTORY = 'acceptance_tests/features'
+DEFAULT_SYSTEM_DIRECTORY = 'system_tests/features'
+DEFAULT_TAGS = '~@standalone'
+
+
+def is_valid_sequential_environment():
+    if os.getenv('IGNORE_SEQUENTIAL_DATA_SETUP') == None:
```

Comment:
`None` checks are best done with `is None` or `is not None` rather than equality check.

Reply:
fixed